### PR TITLE
Add minimal jekyll quickstart; fix bio photo issues; expandable "accordion" for about-sxs/people

### PR DESCRIPTION
**Pull request by duetosymmetry**

### CODE REVIEW

**Comment by duetosymmetry** [comment]
@moble review when you get a chance?

**Comment by duetosymmetry** [comment]
Partially addresses #1

**Comment by markscheel** [comment]
Which version of ruby do you need?  I get errors
```
jekyll 3.9.0 | Error:  undefined method `untaint' for "Tidal effects in binaries involving neutron stars":String
```

**Comment by duetosymmetry** [comment]
My `.ruby-version` has 2.6.10, but I assume others work... I don't know what the github action deployment uses

**Comment by duetosymmetry** [comment]
Now also addressing #3 and #6

File: .github/README.md [hunk 1]
```diff
@@ -10,6 +10,22 @@ github doesn't enable by default.
...
+If you want to test the site locally before submitting a PR, you need
+to have Jekyll and all the dependencies installed.
+
+1. Install ruby through your system's package manager
+2. Optionally install rbenv so you can have more fine-grained control
+   to use a specific version of ruby in just this repo
+3. Do `gem install bundler` to install the ruby "gem" named bundler
+4. Do `bundle install` to install the gems for jekyll and all other
+   dependancies
+5. Do `git submodule update` to make sure you have the files in `_sass/bulma`
```
Comment (markscheel) [inline .github/README.md]: Please add 4.5.  Do `git submodule init`

Comment (duetosymmetry) [reply .github/README.md]: Updated

**Pull request merged**